backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/pickingV2/productsToPick/process/ProductsToPick_4EyesReview_ProcessAll.java

```diff
@@ -76,23 +76,31 @@ protected ProcessPreconditionsResolution checkPreconditionsApplicable()
 			return ProcessPreconditionsResolution.rejectWithInternalReason("only picker shall be allowed to process");
 		}
 
+		final List<ProductsToPickRow> rows = getRowsNotAlreadyProcessed();
```

Comment:
I think you should check if there are **unpacked** rows as well, not only the ones that are not already processed. If the rows were picked but not yet packed, the "Process"  action is available and one can set the shipper transportation and believe the stuff really got in there, when in fact they didn't.

Reply:
If there are Picked rows but not Packed, the Process action does not appear.

![image](https://user-images.githubusercontent.com/4482210/88505146-50e40980-cfdf-11ea-8bee-bb68ac25cccf.png)